### PR TITLE
Create send_file.py

### PR DESCRIPTION
**Pull request by joshuas99**
Adding a script for sending files

### CODE REVIEW

**Comment by joshuas99** [comment]
@gjanco I think the title for this isn't corresponding to our required. Happy to amend if needed. Want this script to be available for anyone in our tutorials.

File: scripts/send_file.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    response = requests.post(api_url, headers={'Authorization': api_key}, data=json.dumps(chunk))
+    json_response = json.loads(response.text)
+    print(json_response)
+    time.sleep(s) # set how long to wait between requests
+    return
+
+def open_file(filename):
+    reader = open(filename, mode='r')
+    for i, line in enumerate(reader):
+        if (i % chunksize == 0 and i > 0):
```
Comment (gjanco) [inline scripts/send_file.py]: Minor, but what if they hit the max 2MB size in the chunk?

**Comment by gjanco** [comment]
Please also add to the Readme how to use this.

**Review by gjanco — CHANGES_REQUESTED** [review]

File: scripts/send_file.py (hunk 1 above)
Comment (joshuas99) [reply scripts/send_file.py]: Ah good point. I can add a check for the size.

Comment (joshuas99) [reply scripts/send_file.py]: Okay, I fixed this by counting the string size as it increments. I also noticed a bug in that I was dropping the lines that happened during the request execution.

**Comment by joshuas99** [comment]
Added the README and updated script

**Comment by joshuas99** [comment]
Noticed I also was mixing conventions on my naming. Fixed that up

**Comment by joshuas99** [comment]
@gjanco is this good to merge?

**Comment by joshuas99** [comment]
Is this good to merge @gjanco ?

**Review by gjanco — APPROVED** [review]

**Pull request merged**